recipes/aerospike-common/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "0009245":
```

Comment:
For these cases, we usually use `cci.YYYYMMDD` with the date of the picked commit